operations/add-bosh-san.yml

```diff
@@ -0,0 +1,69 @@
+---
+- path: /variables/dns_healthcheck_tls_ca
```

Comment:
this (and each of these instances) needs to be changed back to just 
path: /variables/-

Reply:
i lie, needs to be changed to -->
/variables/name=~1<name>